### PR TITLE
MY 005 009 012

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
MY 005 newly reworked, minor changes in two other mss.

### CODE REVIEW

File: ES/ESmy005.xml [hunk 1]
```diff
@@ -30,103 +30,2045 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                            </incipit>
+
+                         </msItem>
+
+
+                     <msItem xml:id="ms_i1.5">
+                      <locus from="8rc" to="108vc"/>
+                      <title type="complete" ref="LIT3585Taamme">Miracles of Mary: 153 miracles</title>
+
                      </msItem>
```
Comment (eu-genia) [inline ES/ESmy005.xml]: nesting error:
this `</msItem>` should be moved to the end of `ms_i1.5`, after all the miracles, before the very last `</msItem>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy005.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy005.xml]: Sorry I do not understand, if you mearn Salāmāt-greetings to John the Evangelist, why should it be after all miracles?

**Comment by eu-genia** [comment]
I mean the nesting is wrong.
Should be
Item 1.5 opens
      Item 1.5.1
      Item 1.5.3
     Item 1.5.4
       Etc etc
     Item 1.5.153 closed
Item 1.5 closed

Now Item 1.5 is closed before 1.5.1

Zitat von DenisNosnitsin1970 ***@***.***>:

> @DenisNosnitsin1970 commented on this pull request.
>
>
>
>> +                            <title>Salāmāt-greetings to John the  
>> Evangelist,
> +                               be read after the Miracles</title>
> +                            <textLang mainLang="gez"/>
> +                            <incipit xml:lang="gez">
> +                            <hi rend="rubric">እምድኅረ፡ ተፍጻሜተ፡  
> ተአምሪሃ፡</hi>፡ ይቤሉ፡ ዘንተ፨
> +                            <hi rend="rubric">ሰላም፡ ለዮሐንስ፡ ወልደ፡</hi>  
> ነጐድጓድ፨ ሰላም፡ ለከ፡ ዮሐንስ፡ ወልደ፡
> +                            አቡቀለምሲስ፨ ሰላም፡ ለከ፡ ዮሐንስ፡ ቴዎሎጎስ፤
> +                            </incipit>
> +
> +                         </msItem>
> +
> +
> +                     <msItem xml:id="ms_i1.5">
> +                      <locus from="8rc" to="108vc"/>
> +                      <title type="complete"  
> ref="LIT3585Taamme">Miracles of Mary: 153 miracles</title>
> +
>                       </msItem>
>
> Sorry I do not understand, if you mearn Salāmāt-greetings to John  
> the Evangelist, why should it be after all miracles?
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BetaMasaheft/Manuscripts/pull/1465#discussion_r813974329
> You are receiving this because your review was requested.
>
> Message ID: ***@***.***>


-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian and Eritrean Studies
Asien Afrika Institut
Universität Hamburg
Alsterterrasse 1
20354 Hamburg
Tel. during the Home Office time +49 163 2879120
***@***.***

https://www.aai.uni-hamburg.de/en/ethiostudies.html

File: ES/ESmy005.xml (hunk 1 above)
Comment (eu-genia) [reply ES/ESmy005.xml]: I mean the nesting is wrong.
Should be
Item 1.5 opens
     Item 1.5.1
     Item 1.5.3
    Item 1.5.4
      Etc etc
    Item 1.5.153 closed
Item 1.5 closed

Now Item 1.5 is closed before 1.5.1

Comment (DenisNosnitsin1970) [reply ES/ESmy005.xml]: Like what I did?

**Pull request merged**